rudder-core/src/main/scala/com/normation/cfclerk/services/AsyncDeployAgent.scala

```diff
@@ -0,0 +1,7 @@
+package com.normation.cfclerk.services
+
+trait AsyncDeployAgent {
+
+  def launchDeployment[T](dest: T) : Unit
```

Comment:
Remove generic and dest should me rename msg and should be of type StartDeploymentMessage